src/api/routes.js

```diff
@@ -22,6 +22,11 @@ var appRouter = function(app) {
 		res.status(200);
 		res.end();
 	})
+
+	app.get("/", function(req , res){
```

Comment:
I was planning / to be a kind of possible routes descriptor.
Like it should list all possible routes supported by supervisor at any given point in time along with the route description like params, response and route description like an rsdl in java applications

Reply:
Sure, we can change the response message for that.
Right now , when I do a GET / . Nothing appeared on the browser. We probably need to use nginx for route descripter page at this route.

Reply:
Ack.
Please make sure to log a issue for the route descriptor part we can triage it based on priority